warehouse/query-core/src/main/java/datawave/query/jexl/visitors/IteratorBuildingVisitor.java

```diff
@@ -564,12 +566,12 @@ public Object visit(ASTNotNode not, Object data) {
     }
     
     private String formatIncludesOrExcludes(List<NestedIterator> in) {
-        String builder = in.toString();
-        builder = builder.replaceAll("OrIterator:", "\n\tOrIterator:");
-        builder = builder.replaceAll("Includes:", "\n\t\tIncludes:");
-        builder = builder.replaceAll("Excludes:", "\n\t\tExcludes:");
-        builder = builder.replaceAll("Bridge:", "\n\t\t\tBridge:");
-        return builder.toString();
+        String s = in.toString();
+        s = s.replace("OrIterator:", "\n\tOrIterator:");
+        s = s.replace("Includes:", "\n\t\tIncludes:");
+        s = s.replace("Excludes:", "\n\t\tExcludes:");
+        s = s.replace("Bridge:", "\n\t\t\tBridge:");
```

Comment:
these are not equivalent

Reply:
I'll add a unit test to prove that these are functionally equivalent. In this case because the first arg isn't an actual regex, it's better to use `String.replace()` [see the sonar explanation for more details](https://rules.sonarsource.com/java/RSPEC-5361)